android/src/main/java/com/applovin/reactnative/AppLovinMAXModule.java

```diff
@@ -81,6 +80,55 @@
     private static final String SDK_TAG = "AppLovinSdk";
     private static final String TAG     = "AppLovinMAXModule";
 
+    private static final String ON_BANNER_AD_LOADED_EVENT     = "OnBannerAdLoadedEvent";
+    private static final String ON_BANNER_AD_LOADFAILED_EVENT = "OnBannerAdLoadFailedEvent";
```

Comment:
I think we should break up the name into separate words - e.g. `ON_BANNER_AD_LOAD_FAILED_EVENT`.